lib/api/liskApi.js

```diff
@@ -539,6 +539,20 @@ LiskAPI.prototype.getAccount = function (address, callback) {
 };
 
 /**
+ * @method generateAccount
+ * @param secret
+ * @param callback
+ *
+ * @return API object
+ */
+
+LiskAPI.prototype.generateAccount = function (secret, callback) {
+	this.sendRequest('accounts/generatePublicKey', { secret: secret }, function (result) {
```

Comment:
Please use the cryptographic function instead of an API call:
https://github.com/LiskHQ/lisk-js/blob/development/lib/transactions/crypto/keys.js